### DISCUSSION TITLE
heartbeat modified date changes when there is no browser connection.

### DISCUSSION THREAD

**Discussion by Oliver-Weng-Sorcerer** [coder/code-server#2108] [Q&A] [answered]
Hi code server community:

I'm implementing a sh file that checks the heartbeat modified date and shut down the VM accordingly.

interestingly that the heartbeat file modified date will change even if there is no browser connection to the code server (I made sure all browser tabs with code server are closed).

wondering if there is anything else that could update the date of the heartbeat file?

**Comment by Oliver-Weng-Sorcerer** [comment]
updates:

based on the latest observation, it seems the heartbeat file Modify date is still updating every minute without any browser connection.
wondering what could cause that?
it may be a bug ?

Thanks

**Comment by nhooyr** [comment, marked as answer]
Likely a bug, glad you opened https://github.com/cdr/code-server/issues/2112 so we can track it there.
